docs/source/configuring.md

```diff
@@ -265,6 +265,13 @@ If you use a version control system besides Git, you can use the environment var
 
 Currently, only Git is fully supported by Apollo Studio.
 
+## Bypass TLS/SSL Validation
+
+In some configurations (often on internal networks) users may need Rover to communicate over encrypted channels (e.g., HTTPS) but avoid the more stringent digital certificate verifications which validate hostnames or may even wish to bypass the digital certificate validation entirely.  This is generally not recommended and considered to be much less secure but for cases where it's necessary, there are two flags you can use to configure how Rover validates HTTPS requests:
```

Comment:
Replace all instances of "may" with "might" where equivalent

Comma after "networks)"

"users may need" -> "you might need"

"which validate hostnames" -> "that validate hostnames." (end sentence)

"or may even wish" -> "You might even need"